Core Classes/DataModel.swift

```diff
@@ -1433,7 +1433,47 @@ final class TimesheetsDataModel: NSObject, AddPilotPopoverDelegate, NSFetchedRes
             statsManager.updateKVSwithTotalNumberOfGliderFlight(totalNumber)
         }
     }
-    
+
+    func getGlidingCentre(forName glidingCentreName : String) -> GlidingCentre
```

Comment:
Is it worth refactoring to reflect that they're called "Flying Sites" now instead of "Gliding Centres"?

Reply:
The idea is good. In the UI, it is called a Unit or Flying Site. The code should effectively follow the same convention.
We can create an issue for that - keep in mind that to do that across all the code, it would include the CloudKit schema as well. Not sure how that would apply (schema migration wise).